addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -1,9 +1,22 @@
 @include media-breakpoint-down(md) {
-    img, .media_iframe_video, span.fa, i.fa {
+    img, .media_iframe_video {
         &:not(.o_animate) {
             transform: none !important;
         }
     }
+
+    // For RTL (Right-to-Left) purposes, don't disable the transform directive for 
+    // Font Awesome icons that are flipped by 180deg.
+    // Exclude elements with classes containing 'fa' and ending with '-left' or '-right'.
+
+    $not-selectors: '';
+    @each $class in $rtl-fa-classes {
+        $not-selectors: '#{$not-selectors}:not(.#{$class})';
+    }
+    #{$not-selectors}:not(.o_animate) {
+        transform: none !important;
+    }
+    
```

Comment:
@SergeBayet This is creating a rule which is only a series of `:not`... :thinking: Probably missing a `.fa` in front of it otherwise it would basically target everything?
![image](https://github.com/odoo/odoo/assets/10338094/90a8aaa2-4295-4b53-92d0-8fdea5e8b086)


Also:
- Hard limit of 80 chars per line for comments
- There is an useless blank line at the end
- Nitpicking: probably don't add a blank line between the comment and the code it comments
- I would remove the last sentence of your comment: it confuses me. I read "Do not do this [...] except for [...]" while this is the exact opposite.

Reply:
@qsm-odoo Ouch :) 
This is solved now...
![image](https://github.com/odoo/odoo/assets/22252071/f628eaa7-377d-43dd-9c6b-6298b3167d95)
![image](https://github.com/odoo/odoo/assets/22252071/ac583690-d471-4060-9d3e-48580f2aaed2)

Reply:
Last nitpicking suggestion @SergeBayet , in your screenshot you can see: 
![image](https://github.com/odoo/odoo/assets/10338094/e892ee9a-cf75-4245-8db7-8b2a88215aeb)

This is kinda bad to have a specificity that high, especially as a stable fix. Let's remove the specificity induced why the `:not`, let's use `:where` around all these `:not` (`.fa:where(:not(.fa-XXX):not(.fa-YYY)...)`)